ecosystem/web-wallet/src/pages/Login.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import KeyInput from "../components/KeyInput";
+import { AptosAccount } from "aptos";
+import { Buffer } from 'buffer'
+import { useNavigate } from "react-router-dom";
+import { useGlobalState } from "../GlobalState";
+
+import './App.css';
+
+export default function Login() {
+  const [, dispatch] = useGlobalState();
+  const navigate = useNavigate();
+
+  function handleSubmitKey(key: string) {
+    let encodedKey = Uint8Array.from(Buffer.from(key, 'hex'))
+    const account = new AptosAccount(encodedKey, undefined);
+    dispatch({account: account});
+    navigate('/wallet');
+  }
+
+  return (
+    <div className="App-header">
+      <h2>Aptos Wallet</h2>
+      <KeyInput submitKey={handleSubmitKey}/>
```

Comment:
If you use a plain `<input>` inside of a `<form>`, you will get a `submit` event when enter is pressed (you could use `<form onSubmit>` to handle it). Just make sure to `preventDefault()` the event so it doesn't reload the page. This way we won't need the `KeyInput` component (and the form will work better with screen readers).